packages/core/test/ovm/Finder.js

```diff
@@ -0,0 +1,58 @@
+/* External Imports */
+const { web3 } = require("hardhat");
+const { didContractThrow } = require("@uma/common");
+const { utf8ToHex, hexToUtf8, toChecksumAddress, randomHex, padRight } = web3.utils;
+const Finder = require("../../artifacts-ovm/contracts/ovm/oracle/implementation/Finder.sol/Finder.json");
```

Comment:
Ideally can use `getAbi` here

Reply:
Ended up using `hardhat.artifacts.readArtifactsSync` which is the canonical hardhat way to get artifacts. I realize that all the methods exported by `core/index.ts` are designed for truffle tests. 

There is no obvious truffle plugin for Optimism, so when running bots eventually like the `optimistic-oracle-proposer` we'll need to consider whether to run them as `truffle` or `hardhat` scripts.